transition-guide/sections/management_of_wis1_and_gts.adoc

```diff
@@ -11,24 +11,24 @@ During the migration to WIS2, the Main Telecommunication Network (MTN), linking
 
 ==== Regional Telecommunication Hubs
 
-Regional Telecommunication Hubs (RTHs) shall keep their Message Switching Systems (MSS) up and continue to publish data Collecting the bulletins from their associated National Meteorological Centres (NMCs) and transmitting them in the appropriate form on the Main Telecommunication Network (MTN), either directly or through the appropriate WMC/RTH in GTS until all Members in their area of responsability migrate from GTS to WIS2.
+Regional Telecommunication Hubs (RTHs) shall keep their Message Switching Systems (MSS) up and continue to publish data Collecting the bulletins from their associated National Meteorological Centres (NMCs) and transmitting them in the appropriate form on the Main Telecommunication Network (MTN), either directly or through the appropriate WMC/RTH in GTS until all Members in their area of responsibility migrate from GTS to WIS2.
 
 When Regional Telecommunication Hubs (RTHs) have migrated to WIS2 Node and all Members in their area of responsibility (AoR) have migrated to WIS2, RTHs may decide to turn off their message switching system (MSS). 
 
-In this case, they will follow the procedure described in xxx.
+In this case, they will follow the procedure described in xxx (TODO: Crossref).
```

Comment:
Note that this is now updated in #42, suggest to remove the addition so that once this PR is merged, PR #42 will update accordingly.